### PR TITLE
Benchmark improvements

### PR DESCRIPTION
**Pull request by kripken**
Normalize too-short scores, handle 0 reps, general cleanup.

### CODE REVIEW

File: tests/test_benchmark.py [hunk 1]
```diff
@@ -72,24 +72,25 @@ def bench(self, args, output_parser=None, reps=TEST_REPS):
...
-
-    print('   %10s: mean: %4.3f (+-%4.3f) secs  median: %4.3f  range: %4.3f-%4.3f  (noise: %4.3f%%)  (%d runs)' % (self.name, mean, std, median, min(self.times), max(self.times), 100 * std / mean, self.reps), end=' ')
-
-    if baseline:
-      mean_baseline = sum(baseline.times) / len(baseline.times)
-      final = mean / mean_baseline
-      print('  Relative: %.2f X slower' % final)
-    else:
-      print()
+    if len(self.times) > 0:
```
Comment (sbc100) [inline tests/test_benchmark.py]: I would just do `if self.times:`

File: tests/test_benchmark.py [hunk 2]
```diff
@@ -487,7 +476,7 @@ def test_memops(self):
         return 0;
       }
     '''
-    self.do_benchmark('memops', src, 'final:', emcc_args=['-s', 'MINIMAL_RUNTIME=1'])
+    self.do_benchmark('memops', src, 'final:', emcc_args=['-s', 'MINIMAL_RUNTIME=0'])
```
Comment (sbc100) [inline tests/test_benchmark.py]: Why this?

Comment (kripken) [reply tests/test_benchmark.py]: MINIMAL_RUNTIME doesn't work with the wasm backend yet. I could remove this entirely, but this does mark the tests that are ready for that mode (i.e. work in that mode with asm2wasm, and hopefully soon with upstream too).

File: tests/test_benchmark.py (hunk 1 above)
Comment (kripken) [reply tests/test_benchmark.py]: thanks, done.

**Pull request merged**